express-test.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const router = express.Router()
+const request = require('request');
+
+router.post('/downlad-url', (req, res) => {
+    downloadURL(req.body.url, () =>{
+        res.send('Done')
+    }) 
+});
+
+const downloadURL = (url, onend) => {
+    const opts = {
+      uri: url,
+      method: 'GET',
+      followAllRedirects: true
+    }
+  
+    request(opts)
+      .on('data', ()=>{})
+      .on('end', () => onend())
+      .on('error', (err) => console.log(err, 'controller.url.download.error'))
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-low.svg" width="20"/></sub>&nbsp; Use of Externally-Controlled Format String

Unsanitized user input from `the HTTP request body` flows into `log`, where it is used as a format string. This may allow a user to inject unexpected content into an application log.

<h6>
Line 21 | <a href="https://cwe.mitre.org/data/definitions/134.html">CWE-134</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 402</a> | <a href="https://learn.snyk.io/lesson/no-rate-limiting/">Learn more about this vulnerability</h6>

<details>

<summary><b>Data flow: 15 steps</b></summary>

`Step 1 - 3` https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/express-test.js#L6

`Step 4` [express-test.js#L11](https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/express-test.js#L11)

`Step 5 - 6` [express-test.js#L13](https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/express-test.js#L13)

`Step 7` [express-test.js#L12](https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/express-test.js#L12)

`Step 8 - 9` [express-test.js#L18](https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/express-test.js#L18)

`Step 10` [express-test.js#L19](https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/express-test.js#L19)

`Step 11` [express-test.js#L18](https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/express-test.js#L18)

`Step 12` [express-test.js#L20](https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/express-test.js#L20)

`Step 13 - 15` https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/express-test.js#L21

</details>